src/pages/index.js

```diff
@@ -212,54 +42,366 @@ const IndexPage = () => (
 
       <SprkStackItem>
         <div className="docs-o-Grid docs-o-Grid--home-page docs-o-Grid--large">
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserDesigners}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserDevelopers}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserComponents}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserGuides}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserFoundations}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserUtils}
-          />
-
-          <SprkCard
-            additionalClasses="docs-c-Card"
-            variant="teaser"
-            teaserConfig={teaserNews}
-          />
+          <SprkCard additionalClasses="docs-c-Card">
+            <SprkStackItem>
+              <SprkLink
+                variant="unstyled"
+                href="/principles/design-principles"
+                analyticsString="teaser-media"
+              >
+                <img
+                  className="sprk-c-Card__media"
+                  alt="Go to Designer Basics"
+                  src={designersMedia}
+                />
+              </SprkLink>
+            </SprkStackItem>
+
+            <SprkStack
+              additionalClasses="sprk-o-Stack__item sprk-c-Card__content"
+              itemSpacing="large"
+            >
+              <SprkStackItem>
+                <SprkHeading element="h3" variant="displayFive">
+                  Designer Basics
+                </SprkHeading>
+              </SprkStackItem>
+
+              <SprkStackItem>
+                <SprkText variant="bodyTwo">
+                  Get started with Design by reading the principles that we use
+                  that we use in Spark Design System.
+                </SprkText>
+              </SprkStackItem>
+
+              <SprkStack
+                additionalClasses="
+                  sprk-o-Stack__item
+                  sprk-o-Stack--end-column"
+              >
+                <SprkStackItem>
+                  <SprkLink
+                    variant="unstyled"
+                    additionalClasses="sprk-c-Button sprk-c-Button--secondary"
+                    href="/principles/design-principles"
```

Comment:
We had a hotfix request come in last week that they had me do on the `main` branch but we havent updated staging yet. Can you pull in those changes here? You'll need them because the `/principles/design-principle` page doesn't exist anymore and it has this linking to a different page.